scripts/uberenv/packages/serac/package.py

```diff
@@ -123,7 +130,7 @@ class Serac(CMakePackage, CudaPackage):
     # Libraries that do not have a debug variant
     depends_on("conduit@master~shared~python")
     depends_on("caliper@master~shared+mpi~callpath~adiak~papi", when="+caliper")
-    depends_on("superlu-dist@5.4.0~shared")
+    depends_on("superlu-dist@6.1.1~shared")
```

Comment:
Can we turn the superlu tests back on with this change?

Reply:
I have some changes to the SuperLU in an MFEM branch that probably should be handled first, but I was hoping that they would instead be permanently replaced by Pastix or something else that's more stable.